torch/_inductor/compile_fx.py

```diff
@@ -1377,10 +1377,11 @@ def codegen_and_compile(
                                 compiled_module, "runner", None
                             )
 
-                    num_bytes, nodes_num_elem, node_runtimes = graph.count_bytes()
-                    metrics.num_bytes_accessed += num_bytes
-                    metrics.node_runtimes += node_runtimes
-                    metrics.nodes_num_elem += nodes_num_elem
+                    if config._metrics_log_runtime:
```

Comment:
Should we just move this to TORCH_LOGS="inductor_metrics" ? it's a small change to add the logs - see https://github.com/pytorch/pytorch/pull/147248/files